src/index.json

```diff
@@ -10300,6 +10300,60 @@
             }
         ],
         "aksarc": [
+            {
```

Comment:
We suggest you change the version to `1.0.0b1` (preview) or `1.0.0` (stable).

As now CLI extension version should conform to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

Reply:
What is the reason for this? We are releasing an older version of our extension. Are you saying the minimum version has to be 1.0.0?